pynautobot/core/app.py

```diff
@@ -82,16 +82,18 @@ def choices(self):
 
         return self._choices
 
-    def custom_choices(self):
+    def custom_fields(self):
         """Returns custom-fields response from app
 
         :Returns: Raw response from Nautobot's custom-fields endpoint.
         :Raises: :py:class:`.RequestError` if called for an invalid endpoint.
         :Example:
 
-        >>> nb.extras.custom_choices()
-        {'Testfield1': {'Testvalue2': 2, 'Testvalue1': 1},
-         'Testfield2': {'Othervalue2': 4, 'Othervalue1': 3}}
+        >>> custom_fields_list = nb.extras.custom_fields()
+        >>> print(custom_fields_list[0]['label'])
+        Test custom field for rack
+        >>> print(custom_fields_list[0]['content_types'])
+        ['dcim.rack']
         """
         custom_fields = Request(
             base="{}/{}/custom-fields/".format(
```

Comment:
Would be good to convert that to f-string. Refactor is welcomed when it simplifies the code.